x/bank/legacy/v043/store_test.go

```diff
@@ -90,13 +90,13 @@ func TestBalanceKeysMigration(t *testing.T) {
 	err = v043bank.MigrateStore(ctx, bankKey, encCfg.Marshaler)
 	require.NoError(t, err)
 
-	newKey := append(types.CreateAccountBalancesPrefix(addr), []byte(fooCoin.Denom)...)
+	newKey := types.CreatePrefixedAccountStoreKey(addr, []byte(fooCoin.Denom))
```

Comment:
IMO the caller shouldn't have to cast into a byte slice -- have `CreatePrefixedAccountStoreKey` do it for them :)

Reply:
Agreed, but should we keep this consistent with upstream to ease future backports / rebases?

Reply:
This is just test shouldn't matter much right?

Reply:
Is this what core SDK does? Sigh...